test/e2e/helpers.js

```diff
@@ -346,6 +346,61 @@ const completeCreateNewWalletOnboardingFlow = async (driver, password) => {
   await driver.clickElement('[data-testid="pin-extension-done"]');
 };
 
+const completeCreateNewWalletOnboardingFlowWithMetricsEnabled = async (
```

Comment:
Could this be refactored? It looks very similar to `completeCreateNewWalletOnboardingFlow` with the exception of agreeing to metrics rather than opting out

Reply:
I could add a new flag with an if statement? I was trying to stick to not doing that as you had mentioned in a previous PR that an if statement increased the complexity of it. Is that a design goal?

Reply:
Could we explore extracting the internals of `completeCreateNewWalletOnboardingFlow` into multiple methods?
That way we could reuse the majority of it. 
Currently, this method does a lot of things,  e.g. creating a password could be a single method, same with revealing and confirming SRP

Reply:
Take a look at what i did in the last commit and see if that's what you were thinking.

Reply:
Yes, that's what I was thinking, so we can now create `completeCreateNewWalletOnboardingFlow` and `completeCreateNewWalletOnboardingFlowWithMetricsEnabled` by just replacing the one method `onboardingChooseMetametricsOption`.

Look good!

Reply:
in this PR I replaced the completeCreateNewWalletOnboardingFlowWithMetricsEnabled by just using each of the steps in my new test, while keeping the original completeCreateNewWalletOnboardingFlow which calls the methods in order

Reply:
That works too